addons/board/static/tests/dashboard_tests.js

```diff
@@ -544,4 +544,49 @@ QUnit.test('subviews are aware of attach in or detach from the DOM', function (a
     form.destroy();
 });
 
+QUnit.test("Views should be loaded in the user's language", function (assert) {
+    assert.expect(2);
+
+    var form = createView({
+        View: FormView,
+        model: 'board',
+        data: this.data,
+        session: {user_context: {lang: 'fr_FR'}},
+        arch: '<form string="My Dashboard">' +
+                '<board style="2-1">' +
+                    '<column>' +
+                        '<action context="{\'lang\': \'en_US\'}" view_mode="list" string="ABC" name="51" domain="[]"></action>' +
+                    '</column>' +
+                '</board>' +
+            '</form>',
+        mockRPC: function (route, args) {
+            if (route === "/web/dataset/search_read") {
+                assert.deepEqual(args.context, {lang: 'fr_FR'},
+                    'The data should be loaded with the correct context');
+            }
+
+            if (route === '/web/action/load') {
+                return $.when({
+                    res_model: 'partner',
+                    views: [[4, 'list']],
+                });
+            }
+            return this._super.apply(this, arguments);
+        },
+        archs: {
+            'partner,4,list':
+                '<list string="Partner"><field name="foo"/></list>',
+        },
+
+        interceptsPropagate: {
```

Comment:
no need for this: you can simply use intercepts

Reply:
nevermind

Reply:
you can use testUtils.intercept directly

Reply:
@aab-odoo 
I'm afraid I can't, since the event will be (and actually is) blocked, and the views are not loaded (I tried it...)

Reply:
Indeed :-)